src/main/scala/temple/builder/DockerfileBuilder.scala

```diff
@@ -8,10 +9,29 @@ import temple.generate.docker.ast.{DockerfileRoot, Statement}
 /** Construct a Dockerfile from a Templefile service */
 object DockerfileBuilder {
 
-  def createServiceDockerfile(serviceName: String, service: AbstractServiceBlock, port: Int): DockerfileRoot = {
+  def createServiceDockerfile(
+    serviceName: String,
+    service: AbstractServiceBlock,
+    port: Int,
+    provider: Option[Provider],
+  ): DockerfileRoot = {
     val language    = service.lookupMetadata[Metadata.ServiceLanguage].getOrElse(ProjectConfig.defaultLanguage)
     val dockerImage = ProjectConfig.dockerImage(language)
 
+    // If using DockerCompose and this is the auth service, change the exec
+    val execStatements: Option[Seq[Statement]] = service match {
+      case AbstractServiceBlock.AuthServiceBlock =>
+        provider.collect {
+          case Provider.DockerCompose =>
+            Seq(
+              Run("apk", Seq("add", "curl")),
+              Run("chmod", Seq("+x", "wait-for-kong.sh")),
+              Cmd("./wait-for-kong.sh", Seq("kong:8001", "--", "./auth")),
```

Comment:
Is Kong always on 8001, or did we pick that

Reply:
Always 😄